relay-filter/src/browser_extensions.rs

```diff
@@ -39,7 +39,8 @@ static EXTENSION_EXC_VALUES: Lazy<Regex> = Lazy::new(|| {
         # See: https://forum.sentry.io/t/error-in-raven-js-plugin-setsuspendstate/481/
         plugin\.setSuspendState\sis\snot\sa\sfunction|
         # Chrome extension message passing failure
-        Extension\scontext\sinvalidated
+        Extension\scontext\sinvalidated|
+        webkit-masked-url:
```

Comment:
Do we want to have also `hidden` in there as requested in the linked issue? Or having only `webkit-masked-url` should be fine?